doc/source/cluster/kubernetes/user-guides/static-ray-cluster-without-kuberay.md

```diff
@@ -0,0 +1,162 @@
+(deploy-a-static-ray-cluster-without-kuberay)=
+
+# (Advanced) Deploying a static Ray cluster without KubeRay
+
+This deployment method for Ray no longer requires the use of CustomResourceDefinitions (CRDs).
+In contrast, the CRDs is a prerequisite to use KubeRay. One of its key components, the Kuberay operator,
+manages the Ray cluster resources by watching for Kubernetes events (create/delete/update).
+Although the KubeRay operator can function within a single namespace, the use of CRDs has a cluster-wide scope.
+If the necessary Kubernetes admin permissions are not available for deploying KubeRay, this doc introduces a way to deploy a static Ray cluster to Kubernetes without using KubeRay. However, it should be noted that this deployment method lacks the built-in
+autoscaling feature that KubeRay provides.
+
+## Deploying a static Ray cluster
+
```

Comment:
Maybe we can add some instructions about Kubernetes cluster creation (e.g. `kind create cluster`) to enable users without Kubernetes knowledge to reproduce this doc.

Reply:
Done, added a preparation section.